extensions/toggl-track/src/index.tsx

```diff
@@ -15,13 +15,25 @@ function ListView() {
   const { isLoading, isValidToken, projectGroups, runningTimeEntry, timeEntries, projects } = useAppContext();
   const getProjectById = (id: number) => projects.find((p) => p.id === id);
 
-  const timeEntriesWithUniqueProjectAndDescription = timeEntries.reduce((acc, timeEntry) => {
-    const existing = acc.find((t) => t.description === timeEntry.description && t.pid === timeEntry.pid);
-    if (!existing) {
-      acc.push(timeEntry);
-    }
-    return acc;
-  }, [] as TimeEntry[]);
+  const timeEntriesWithUniqueProjectAndDescription = timeEntries
+    .reduce((acc, timeEntry) => {
+      const existing = acc.find((t) => t.description === timeEntry.description && t.pid === timeEntry.pid);
+      if (!existing) {
+        acc.push(timeEntry);
+      }
+      return acc;
+    }, [] as TimeEntry[])
+    .sort((a, b) => {
```

Comment:
Actually, this is wrong. It should be sorted first and then filtered. I'll update it later today.

Reply:
[`4efb10c` (#4450)](https://github.com/raycast/extensions/pull/4450/commits/4efb10c0cb5159318f7bfd9b56a6627ba5ded1e0)